Airports/EGUW/Positions.txt

```diff
@@ -1,3 +1,3 @@
 EGUW_APP:Wattisham Approach:125.800:UWR:A:EGUW:APP:-:-:4501:4547:N052.07.37.000:E000.57.28.000::
 EGUW_P_APP:Wattisham Talkdown:123.300:UWP:P:EGUW:APP:-:-:4501:4547:N052.07.37.000:E000.57.28.000::
-EGUW_TWR:Wattisham Tower:122.100:UWT:T:EGUW:TWR:-:-:4501:4547:N052.07.37.000:E000.57.28.000::
\ No newline at end of file
+EGUW_TWR:Wattisham Tower:118.210:UWT:T:EGUW:TWR:-:-:4501:4547:N052.07.37.000:E000.57.28.000::
```

Comment:
```suggestion
EGUW_TWR:Wattisham Tower:118.200:UWT:T:EGUW:TWR:-:-:4501:4547:N052.07.37.000:E000.57.28.000::
```
@rishab-alt it's been agreed that it should be the 25KHz variant of the freq for now :)